tests/acceptance/in-repo-addon-destroy-test.js

```diff
@@ -0,0 +1,163 @@
+'use strict';
+
+var Promise              = require('../../lib/ext/promise');
+var conf                 = require('ember-cli-internal-test-helpers/lib/helpers/conf');
+var ember                = require('../helpers/ember');
+var fs                   = require('fs-extra');
+var path                 = require('path');
+var remove               = Promise.denodeify(fs.remove);
+var root                 = process.cwd();
+var tmproot              = path.join(root, 'tmp');
+var Blueprint            = require('../../lib/models/blueprint');
+var BlueprintNpmTask     = require('ember-cli-internal-test-helpers/lib/helpers/disable-npm-on-blueprint');
+var mkTmpDirIn           = require('../../lib/utilities/mk-tmp-dir-in');
+
+var chai = require('../chai');
+var expect = chai.expect;
+var file = chai.file;
+var dir = chai.dir;
+
+describe('Acceptance: ember destroy in-repo-addon', function() {
+  this.timeout(20000);
+
+  before(function() {
+    BlueprintNpmTask.disableNPM(Blueprint);
+    conf.setup();
+  });
+
+  after(function() {
+    BlueprintNpmTask.restoreNPM(Blueprint);
+    conf.restore();
+  });
+
+  beforeEach(function() {
+    return mkTmpDirIn(tmproot).then(function(tmpdir) {
+      process.chdir(tmpdir);
+    });
+  });
+
+  afterEach(function() {
+    process.chdir(root);
+    return remove(tmproot);
+  });
+
+  function initApp() {
```

Comment:
Understood, I'll remove these tests and modify `tests/unit/blueprints/in-repo-addon-test.js` to test the changes of this PR